util/check_dif_statuses.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""DIF Status Report Generator
+
+This tool generates a status report for the DIFs by cross referencing the git
+commit history of each DIF with that of the HW it actuates to provide OpenTitan
+developers with information about what DIFs require updating.
+
+To display usage run:
+    ./check_dif_statuses.py --help
+"""
+
+import argparse
+import collections
+import glob
+import io
+import json
+import logging
+import os
+import re
+import sys
+from contextlib import redirect_stdout
+from enum import Enum
+
+import gitfame
+import hjson
+import pydriller
+from tabulate import tabulate
+from termcolor import colored
+from tqdm import tqdm
+
+# The IP suffix of some DIFs are slightly different than the IP directory name
+# in the hw/ip/ directory, so here is the translation for the applicable IPs.
+IP_TRANSLATION = {
+    "plic": "rv_plic",
+    "entropy": "entropy_src",
+}
```

Comment:
> Is there a good reason for this deviation?
> `rv_timer` retains the `rv_` prefix. Plus, we loosely associate 'entropy' with the whole entropy SS rather than with the `entropy_src` IP. 

Probably not, but other should weigh in on this? 

> IMO it would be better to have the DIF names match the IP names.

Agreed, according to DIF style guide names should match the IP block, so weird that this ended up not matching. We should address the DIF name / IP name mismatch in separate PR.